kws/bin/plot_det_curve.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2021 Binbin Zhang(binbzha@qq.com)
+#                    Menglong Xu
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def load_stats_file(stats_file):
+    values = []
+    with open(stats_file, 'r', encoding='utf8') as fin:
+        for line in fin:
+            arr = line.strip().split()
+            threshold, fa_per_hour, frr = arr
+            values.append([float(fa_per_hour), float(frr)])
+    values.reverse()
+    return np.array(values)
+
+
+def plot_det_curve(keywords, stats_dir, figure_file):
+    plt.figure(dpi=200)
+    plt.rcParams['xtick.direction'] = 'in'
+    plt.rcParams['ytick.direction'] = 'in'
+    plt.rcParams['text.usetex'] = True
+    plt.rcParams['font.size'] = 12
+
+    for index, keyword in enumerate(keywords):
+        stats_file = os.path.join(stats_dir, 'stats.' + str(index) + '.txt')
+        values = load_stats_file(stats_file)
+        plt.plot(values[:, 0], values[:, 1], label=keyword)
```

Comment:
how about the color? is it the same color for multiple wakeup words?

Reply:
for multiple wake-up words (more than one curve), matplotlib will set each a different color by default.
![image](https://user-images.githubusercontent.com/32296227/146323806-c8f02fee-b18c-45ca-a6a9-f6d4f0d1e657.png)